File: jats-demo-webapp/src/main/ts/model/Category.ts
class Category
{
	private id : number;

	private products : Component[];

	public getId() { return this.id; }
	public setId( id : number ) { this.id = id; }

	public getProducts() { return this.products; }
	public setProducts( products : Component[] ) { this.products = products; }


}

File: jats-demo-model/src/main/java/com/github/pepe79/jats/demo/model/base/Category.java
package com.github.pepe79.jats.demo.model.base;

import java.util.List;

public class Category {

	private Integer id;

	private List<Component> products;

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public List<Component> getProducts() {
		return products;
	}

	public void setProducts(List<Component> products) {
		this.products = products;
	}

}
